### DISCUSSION TITLE
Problem running patch based model

### DISCUSSION THREAD

**Discussion by CGuilfoyle** [RangeShifter/RangeshiftR-tutorials#29] [Q&A] [answered]
I am trying to run a patch based model in Rangeshifter GUI. There are 66 patches in the landscape and I am trying to run simulations where the 'All suitable patches' option is selected in the initialization settings - i.e. each simulation begins with all patches (66) being occupied at a specified density. 

However, when I run the model, the number of initial patches occupied is much lower than 66 and varies between each iteration - usually around 20-25. 

Does anyone know why this would be happening ? I have also tried using the 'Manually selected patches' option and input each patch individually but the results are the same. 

Appreciate any help.

**Comment by JetteReeg** [comment, marked as answer]
Hi, 
it sounds a bit that your initial populations die immediately. Have you checked that your density dependence is not leading to a direct mortality of the individuals? Have you tried to decrease the initial density?
Best, Jette

**Reply by CGuilfoyle** [reply]
Hi Jette,
Thanks for the reply. So if I decrease the initial density the initial number of patches seems to decrease (even though I have selected All suitable patches) and when I increase the initial density the number of initial occupied patches increases. The number of occupied patches at the start of each replicate is not consistent and changes randomly. 

An issue which may be linked to the problem is that when I choose the 'manually select patches' option in initialisation, there are certain patches that are not being recognised. E.g. I select patch number 5 for initialisation, which results in 0 occupied patches and thus 0 for population size for all replicates, even though I can see this patch in the landscape visualizer. 

Hope this makes sense,
Colin

**Reply by damariszurell** [reply]
The latter issue sounds like maybe something in the ImportedLandscape() does not match up, e.g. the "K_or_DensDep" vector does not match the habitats in the "LandscapeFile" argument or the "PatchFile" argument.
